packages/composables/src/useMedia/useMedia.ts

```diff
@@ -0,0 +1,35 @@
+import { useShopwareContext } from "#imports";
+import type { operations } from "#shopware";
+
+interface UseMedia {
+  /**
+   * Fetches media by ids
+   * @param {string[]} ids
+   */
+  fetchMedia(
+    ids: string[],
+  ): Promise<operations["readMedia post /media"]["response"]>;
+}
+
+/**
+ *
+ * Composable for media handling
+ *
+ * @returns {UseMedia} media composable
+ */
+export function useMedia(): UseMedia {
```

Comment:
I would like to see what kind of media this composable is made for. Maybe we could add a parameter in type `Media` for composable function, and then we could extract ids from argument and replace/merge it with `ids` provided in `fetchMedia` function which could then become optional. 

the second option to tell something more about the purpose is providing an example in functions' description and changeset file.